src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -840,8 +864,6 @@ TEST_F(HMICapabilitiesTest,
   hmi_capabilities_->Init(last_state_wrapper_);
 
   // Check system capabilities; only phone capability is available
-  EXPECT_FALSE(hmi_capabilities_->navigation_supported());
-  EXPECT_TRUE(hmi_capabilities_->phone_call_supported());
```

Comment:
@RomanReznichenkoLuxoft why did you removed that?

Reply:
done